services/src/test/java/org/jboss/windup/web/services/data/ServiceConstants.java

```diff
@@ -5,6 +5,6 @@
  */
 public abstract class ServiceConstants
 {
-    public static final String REST_BASE = "rest";
-    public static final String FURNACE_REST_BASE = "rest-furnace";
+    public static final String REST_BASE = "";
+    public static final String FURNACE_REST_BASE = "api";
```

Comment:
1) Isn't it weird to have HOST/rhamt-web/api/api ?  
2) Doesnt these two clash?
HOST/rhamt-web/api/*    
HOST/rhamt-web/api/api/*
Or maybe I don't understand the changes.